src/map/map.js

```diff
@@ -217,6 +222,12 @@ class MapModule extends React.Component {
         }
       }
     }
+
+    if (this.props.cursor !== undefined) {
```

Comment:
As we have the default prop marked as `null` this sentence is useless... so what about ?
1. componentDidMount sets setCursor without any check to have the store initialized
2. componentDidUpdate checks if cursor !== null && !== prevCursor to dispatch the change

Reply:
Good catch, thanks!